compute/apiv1/smoke_test.go

```diff
@@ -92,16 +88,12 @@ func TestCreateGetPutPatchListInstance(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	waitZonalRequest := &computepb.WaitZoneOperationRequest{
-		Project:   projectId,
-		Zone:      defaultZone,
-		Operation: insert.Proto().GetName(),
-	}
-	_, err = zonesClient.Wait(ctx, waitZonalRequest)
+	err = insert.Wait(ctx)
+	defer ForceDeleteInstance(ctx, name, c)
 	if err != nil {
 		t.Error(err)
+		t.FailNow()
```

Comment:
nit: there are a couple of these spots, they can be summarized as `t.Fatal(err)`

Reply:
Haha sorry had auto-merge on. I will follow up :)